lib/dashboards/rpc-providers-widgets-factory.ts

```diff
@@ -19,7 +19,7 @@ const ID_TO_PROVIDER = (id: ChainId): string => {
       return ProviderName.INFURA
     case ChainId.CELO:
     case ChainId.BNB:
-    case ChainId.BASE:
+      // case ChainId.BASE:
```

Comment:
Since we are not using quicknode, I thought that would be better to exclude Base from this, lmk if this should be reverted, @msvstj